frontend/src/views/WheresMyGene/components/GeneSearchBar/index.tsx

```diff
@@ -121,13 +119,13 @@ export default function GeneSearchBar({
           analyticsEvent={EVENTS.WMG_SELECT_GENE}
         />
         {downloadFeat && (
-          <>
-            <Button onClick={copyGenes}>Copy Genes</Button>
-            <SaveImage
-              selectedTissues={selectedTissues}
-              selectedGenes={selectedGenes}
-            />
-          </>
+          <SaveImage
+            {...{
+              selectedCellTypes,
+              selectedGenes,
+              selectedTissues,
+            }}
```

Comment:
spreading object disables prop type checking, so I think we should do explicit prop passing!

here and throughout
```tsx
<SaveImage
  selectedCellTypes={selectedCellTypes}
  ...
/>
```